game/civ5map.ksy

```diff
@@ -0,0 +1,157 @@
+meta:
+  id: civ5map
+  application: Sid Meier's Civilization V
+  file-extension: civ5map
+  encoding: UTF-8
+  license: CC0-1.0
+  endian: le
+doc: |
+  Many of Civilization 5's maps are in civ5map format, and encodes terrain and
+  sometimes resources on the map plots. Scenario data is also encoded in some
+  civ5map files, although this does not currently parse such data.
+
+  Civ5maps can be accompanied by lua files which defines advanced behavior for
+  world gen on top of what's already in the civ5map.
+
+  There are three versions of the format identified in this file- A, B, and
+  C. The latest version of Civ can read all of them. Version A is the base
+  version. Version B has some additional information about the map in the
+  header (string3). Version C has an additional part in the header that
+  defines which lua files Civ should use when starting a new game with the map.
```

Comment:
Should https://forums.civfanatics.com/threads/map-index.393594/ be added as a link to examples into the docs?

Reply:
It's a list of fanmade maps, but it hasn't been updated for ten years. I don't know how helpful it would be.

Reply:
The use is to get a link where a reader can fetch some files for testing the spec.

Reply:
I'm still skeptical about linking to non-official maps, since their integrity can't be guaranteed. However, I've included file paths to official Firaxis civ5maps of various versions relative to the Civ 5 install path, which should serve this purpose.